content/content.index.php

```diff
@@ -108,7 +108,8 @@ public function view()
         // Assemble the table
         $table = Widget::Table(
             Widget::TableHead($thead), null,
-            Widget::TableBody($tbody), null
+            Widget::TableBody($tbody), 'selectable', null,
+            array('role' => 'directory', 'aria-labelledby' => 'symphony-subheading', 'data-interactive' => 'data-interactive')
         );
         $table->setAttribute('class','selectable');
```

Comment:
I think you can delete this line because you already set the class in the widget call.